functions_assignment.js

```diff
@@ -0,0 +1,240 @@
+// # Function Exercises
```

Comment:
Overall good job, I see your developing your skills on functions and scope. Chat with me if you would like to go over the answers you are still unsure about. Ill be ore than happy to help guide you!